packages/application-generic/src/factories/sms/sms.factory.ts

```diff
@@ -44,7 +45,12 @@ export class SmsFactory implements ISmsFactory {
   getHandler(integration: IntegrationEntity) {
     const handler =
       this.handlers.find((handlerItem) =>
-        handlerItem.canHandle(integration.providerId, integration.channel)
+        handlerItem.canHandle(
```

Comment:
I think this is the best place to handle it now days. It was not here before because then we used the stateless library instead and that one was much more used by others.